packages/monaco-editor-wrapper/src/editorClassic.ts

```diff
@@ -15,41 +15,54 @@ export type MonacoLanguageExtensionConfig = {
     mimetypes?: string[];
 }
 
-export type EditorClassicConfig = {
+export type EditorAppConfigClassic = {
+    editorAppType: 'classic';
     languageExtensionConfig?: MonacoLanguageExtensionConfig;
     languageDef?: languages.IMonarchLanguage;
     themeData?: editor.IStandaloneThemeData;
 }
 
-export class EditorClassic extends MonacoEditorBase implements MonacoEditorWrapper {
+export class EditorAppClassic extends EditorAppBase implements MonacoEditorWrapper {
```

Comment:
Justification for the name change?

Reply:
monaco-editor itself has an editor and a diffEditor. I wanted to make this more distinguishable. It is very possible that these names are not the best. I will think about it once more.

Reply:
Will come with next update